tests/orderly/test_run.py

```diff
@@ -344,29 +338,29 @@ def test_can_use_shared_resources_directory(tmp_path):
     assert {el.path for el in meta.files} == {
         "shared_dir.py",
         "result.txt",
-        join("shared_data", "numbers.txt"),
-        join("shared_data", "weights.txt"),
+        "shared_data/numbers.txt",
+        "shared_data/weights.txt",
     }
     assert meta.custom == {
         "orderly": {
             "role": unordered(
                 [
                     {"path": "shared_dir.py", "role": "orderly"},
                     {
-                        "path": join("shared_data", "weights.txt"),
+                        "path": "shared_data/weights.txt",
                         "role": "shared",
                     },
                     {
-                        "path": join("shared_data", "numbers.txt"),
+                        "path": "shared_data/numbers.txt",
                         "role": "shared",
                     },
                 ]
             ),
             "artefacts": [],
             "description": {"display": None, "long": None, "custom": None},
             "shared": {
-                join("shared_data", "numbers.txt"): join("data", "numbers.txt"),
-                join("shared_data", "weights.txt"): join("data", "weights.txt"),
+                "shared_data/numbers.txt": "data/numbers.txt",
+                "shared_data/weights.txt": "data/weights.txt",
```

Comment:
is it worth adding something that checks (perhaps on cleanup of a temporary orderly directory) that we never get backslashes anywhere?

Reply:
This is a great idea, because it will get caught in the implementations at least